packages/ui/src/views/repo/repo-summary/repo-summary.types.ts

```diff
@@ -0,0 +1,20 @@
+export enum SummaryItemType {
+  Folder = 0,
+  File = 1
+}
+
+export interface UserProps {
+  name: string
+  avatarUrl?: string
+}
+
+export interface FileProps {
```

Comment:
Same here, only use `Props` when describing the props of a component.
```suggestion
export interface User {
  name: string
  avatarUrl?: string
}

export interface RepoFile {
```